docs/credentials/credential-sharing.md

```diff
@@ -11,20 +11,21 @@ contentType: howto
 Available on all Cloud plans, and Enterprise self-hosted plans.
 ///
 
-You can share a credential you created with other users in the same n8n workspace as you. The other users can then use the credential in their workflows. They can't access or edit the credential details.
+You can share a credential directly with other users to use in their workflows. Or into a project for all members of a project to use in workflows inside that project. Any users accessing a shared credential won't be able to access or edit the credential details.
 
-Users can share credentials they created. Instance owners, and users with the admin role, can view and share all credentials in the instance. Refer to [Account types](/user-management/account-types/) for more information about owners and admins.
+Users can share credentials they created and project admins can share credentials created inside a project. Instance owners, and users with the instance admin role, can view and share all credentials in the instance. Refer to [Account types](/user-management/account-types/) for more information about owners and admins.
```

Comment:
Also minor tweaks to this sentence as follows:

"Users can share credentials they created and own. Only project admins can share credentials created in and owned by a project. Instance owners and instance admins, can view and share all credentials on an instance. Refer to Account types for more information about owners and admins."